src/main/webapp/app/assessment/assessment-layout/assessment-layout.component.html

```diff
@@ -1,6 +1,4 @@
 <jhi-assessment-header
-    [hideBackButton]="hideBackButton"
-    (navigateBack)="navigateBack.emit()"
```

Comment:
Related In/Outputs still exist in `assessment-layout.component.ts`

Reply:
Thanks for the comment, I re-added `navigateBack` because it does indeed have dependencies that are still needed and cannot be removed.

Reply:
Let me know if it's okay and then I can resolve this converstation